cluster/images/etcd/Makefile

```diff
@@ -26,10 +27,10 @@
 # Except from etcd-$(version) and etcdctl-$(version) binaries, we also
 # need etcd and etcdctl binaries for backward compatibility reasons.
 # That binary will be set to the last version from $(BUNDLED_ETCD_VERSIONS).
-BUNDLED_ETCD_VERSIONS?=2.2.1 2.3.7 3.0.17 3.1.12 3.2.24 3.3.10
+BUNDLED_ETCD_VERSIONS?=2.2.1 2.3.7 3.0.17 3.1.12 3.2.24 3.3.17 3.4.4
```

Comment:
why 3.3.17?

Reply:
It is to build k8s.gcr.io/etcd docker images. The etcd versions are the SemVer of latest etcd version available for backward compatibility.

Reply:
Updated to 3.4.4